core/src/main/java/io/undertow/server/handlers/SameSiteCookieHandler.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
```

Comment:
we have different licensed headers in the files. This one has the wrong year.

Reply:
Fixed.